### ISSUE TITLE
Connect to local DB without broker

### ISSUE THREAD

**Issue by gquerret** [BalticAmadeus/ProBro#392]
Is it possible to browser local RO databases ? I tried to use ProBro with this connection in openedge-project.json:

```json
    {
      "name": "sp2k",
      "connect": "-db target/db/sports2000 -ld sp2k -RO",
      "dumpFile": "dump/sports2000.df",
      "aliases": [
        "sports"
      ]
    }
```

The extension complains about no port provided for connection. Is it possible to use read-only or shared-memory connection ?

**Comment by PauliusKu** [comment]
Hi @gquerret, I tested with -RO connection parameter and it worked just fine for me:
This was my DB configuration in `openedge-project.json` file:
```
    {
      "name": "mydb_RO",
      "connect": "-db C:/DB/mydb -ld sp2k -RO",
      "dumpFile": "",
      "aliases": [
        "sports"
      ]
    }
```
And I can connect to it 
![image](https://github.com/BalticAmadeus/ProBro/assets/36190136/9eee80d4-37b0-4486-be12-fcf105e9f3a0)

Can you try to setup it again? And if you still have this issue, can you provide more details?

**Comment by gquerret** [comment]
Tried again with more or less the same setup (reinstalled the latest version of the extension). Started VSCode with `code .` in my project directory, so I expect the work dir to the project dir. I'm always getting this message when I switch to the ProBro view:
![image](https://github.com/BalticAmadeus/ProBro/assets/6498723/0002eb86-9e15-4226-9c82-4e371ba643cf)

If I select `default`, then it shows:
![image](https://github.com/BalticAmadeus/ProBro/assets/6498723/c6e72cab-f1d2-454d-bc5b-e960794b4a49)

The DB connection string is `-db db/sp2k -RO` ; that should be relative to the project directory, not to the extension basedir.

**Comment by PauliusKu** [comment]
Hi @gquerret, yes that's an issue. I will create a separate ticket for that and we'll probably release it soon.
I won't close this ticket until ProBro starts working for you :)

**Comment by PauliusKu** [comment]
@gquerret 
Fix released with version 1.5.1.
Now relative paths in _openedge-project.json_ file configuration are correctly interpreted.

**Comment by PauliusKu** [comment]
Closing, as it was fixed with https://github.com/BalticAmadeus/ProBro/issues/465
